pkg/ui/workspaces/cluster-ui/src/sqlActivity/util.tsx

```diff
@@ -97,7 +97,7 @@ export function filterStatementsData(
         INTERNAL_APP_NAME_PREFIX,
       );
       return (
-        (!appNames?.length && !isInternal) ||
+        !appNames?.length ||
```

Comment:
Why was the `&& isInternal` removed? I thought we wanted to align with the txns page where we only show internal stmts when they are specifically selected in the app name filter.

Reply:
Ah nvm, just read that we're swapping the behaviour here.

Reply:
It's also updated on the Txn page with the new behaviour